src/_guides/language/effective-dart/style.md

```diff
@@ -662,43 +656,43 @@ switch (fruit) {
     break;
 }
 {% endprettify %}
-</div>
 
 
 ### DO indent multi-line method cascades at least two spaces.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (cascade)"?>
 {% prettify dart %}
 buffer
   ..write('Hello, ')
   ..write(name)
   ..write('!');
 {% endprettify %}
-</div>
 
 
 ### PREFER indenting continued lines with at least four spaces.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (four-spaces)"?>
 {% prettify dart %}
-someLongObject.aReallyLongMethodName(longArg, anotherLongArg,
-    wrappedToNextLine);
+someVeryLongVariableName.aReallyLongMethodName(
+    arg, anotherArg, wrappedToNextLine);
```

Comment:
There are two sections using very long identifier names. This was changed to help unify the use of variables names in both sections.